core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.scss

```diff
@@ -0,0 +1,57 @@
+@use "variables" as *;
+
+:host ::ng-deep {
+    .dot-container-properties__title-container {
```

Comment:
This is not need it in the `::ng-deep` because this not need to be outside the angular encap, check the other components as well.

You only need the ::ng-deep when accessing primeng components.

Reply:
This.